serde_derive_internals/lib.rs

```diff
@@ -21,6 +21,7 @@
 extern crate syn;
 
 extern crate proc_macro2;
+extern crate quote;
```

Comment:
Why is an external crate `quote` needed?

Reply:
Hello :smile: 
The `quote` is used in [serde_derive/src/internals/attr.rs#L153](https://github.com/NotBad4U/serde/blob/b127a1703875c85462f072340692ffa022c611eb/serde_derive/src/internals/attr.rs#L153) :
```rust
impl ToTokens for VariantNameType {
    fn to_tokens(&self, out: &mut TokenStream) {
        match *self {
            VariantNameType::Str(ref s) => s.to_tokens(out),
            VariantNameType::Bool(b) => {
                b.to_tokens(out);
            },
            VariantNameType::Int(i) => {
                i.to_tokens(out);
            }
        }
    }
}
```
`serde_derive_internal/` has a link to `./serde_derive/src/internals/` so if we need to `build` `serde_derive_internal` alone that will not compile without the `extern crate quote;`. If the sub crate `serde_derive_internal` is not meant to be built alone I'll remove this dependency.